engine/lib/notification.php

```diff
@@ -139,6 +148,22 @@ function elgg_remove_subscription($user_guid, $method, $target_guid) {
 }
 
 /**
+ * Returns an array of delivery methods used to notify the user about a target entity,
+ * or false if user is not subscribed
+ * 
+ * @param int $user_guid   GUID of the subscriber
+ * @param int $target_guid GUID of the subscription target
+ * @retun array|false
+ */
+function elgg_get_subscription_record($user_guid, $target_guid) {
```

Comment:
This might be more useful if we added an optional third param that would be the notification method to check against.